compiler/rustc_target/src/target_features.rs

```diff
@@ -34,9 +34,6 @@ pub enum Stability {
     /// particular for features are actually ABI configuration flags (not all targets are as nice as
     /// RISC-V and have an explicit way to set the ABI separate from target features).
     Forbidden { reason: &'static str },
-    /// This feature can not be set via `-Ctarget-feature` or `#[target_feature]`, it can only be set
-    /// by target modifier flag. Target modifier flags are tracked to be consistent in linked modules.
-    TargetModifierOnly { reason: &'static str, flag: &'static str },
```

Comment:
This variant was only needed because https://github.com/rust-lang/rust/pull/135927  fed the retpoline flags into the target feature machinery together with `-Ctarget-features`. Things get a lot cleaner if we just keep them separate all the way through.